lifelines/fitters/__init__.py

```diff
@@ -2244,6 +2249,7 @@ def print_summary(self, decimals=2, style=None, columns=None, **kwargs):
         footers.extend(
             [
                 ("AIC", "{:.{prec}f}".format(self.AIC_, prec=decimals)),
+                ("BIC", "{:.{prec}f}".format(self.BIC_, prec=decimals)),
```

Comment:
I like everything, but am unsure about this. I don't want print_summary to be overloaded with metrics / information. Already AIC is there - does having BIC there add much value to the user? What are your thoughts?